kdestandardize.ado

```diff
@@ -179,8 +179,21 @@ prog def kdestandardize
 					   6 `"Percentage of Teacher Turnover"'                  ///
 					   7 `"School Leadership Composite"', modify
 
+	   // Define EQ_LABEL
+	   la def eqlabel 1`"Exemplary/Accomplished"'                            ///
+					  2`"High/Expected"'                                     ///
+					  3`"School-Level"'                                      ///
+					  4`"Strongly Agree/Agree"'                              ///
+					  5`" "', modify
+					  
+	   // Define PROGRAM_TYPE
```

Comment:
Use `progtype` for the variable name here.  It might make more sense to denormalize the data associated with these categories, but will have to review further.